jenkins/node-image/node-runner.sh

```diff
@@ -27,6 +27,5 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
 fi
 
-
 # Run script
-${NODE_SCRIPT} ${NODE_PROPERTY}
+"${NODE_SCRIPT:-test/e2e_node/jenkins/e2e-node-jenkins.sh} ${NODE_PROPERTY:-test/e2e_node/jenkins/jenkins-ci.properties}"
```

Comment:
this is not the fix you want.

it looks like the issue is that in `node-dockerized.sh` and `ci-kubernetes-node-kubelet-dockerized.sh` the env var is named `NODE_PROPERTIES`, not `NODE_PROPERTY`.

please fix that here and remove the defaults.